bindata/bootkube/manifests/00_etcd-endpoints-cm.yaml

```diff
@@ -2,6 +2,10 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: etcd-endpoints
-  namespace: openshift-etcd
+  namespace: openshift-etcd {{ if not .BootstrapScalingStrategy.InPlace }}
   annotations:
     alpha.installer.openshift.io/etcd-bootstrap: {{ .BootstrapIP }}
+{{ end }} {{ if .BootstrapScalingStrategy.InPlace }}
+data:
+  {{ .EtcdEndpointConfigmapData }}
```

Comment:
(No action required) Note that it's possible to do base64 encoding in the template:


```yaml
data:
  # Use base64-encoded ip address as the key to emulate endpoint controller
  {{ .BootstrapIP | base64 }}: {{ .BootstrapIP }}
```

Reply:
nice trick, sold

Reply:
I see KAS-O does this with the load command but inline it fails. Since it works I am inclined to leave it as is and research for further optimization.

```
 <base64>: wrong type for value; expected []uint8;
```

Reply:
maybe when you gut the stuff we don't need in render we can add sprig which has some nice helpers.

https://github.com/Masterminds/sprig

Reply:
adding TODO